samples/apps/copilot-chat-app/webapi/SemanticKernelExtensions.cs

```diff
@@ -25,11 +24,6 @@ internal static class SemanticKernelExtensions
     /// </summary>
     public delegate Task RegisterSkillsWithKernel(IServiceProvider sp, IKernel kernel);
 
-    /// <summary>
-    /// Delegate to register skills with a planner.
-    /// </summary>
-    public delegate Task RegisterSkillsWithPlanner(IServiceProvider sp, CopilotChatPlanner planner);
```

Comment:
This looks like it should be left in there!

Reply:
This is not used.

Reply:
@adrianwyatt - Wasn't this part of your effort to genericize the loading of skills?

Reply:
It was, but we didn't end up using. Let's remove it.